Dapper.Contrib/SqlMapperExtensions.cs

```diff
@@ -418,26 +418,35 @@ private static string GetTableName(Type type)
         /// Specifies a custom callback that detects the database type instead of relying on the default strategy (the name of the connection type object).
         /// Please note that this callback is global and will be used by all the calls that require a database specific adapter.
         /// </summary>
-        public static GetDatabaseTypeDelegate GetDatabaseType;
-
-
-        private static ISqlAdapter GetFormatter(IDbConnection connection)
+        [Obsolete("Set Dapper.Config.GetDbmsTypeFromConnection instead, this gets used for more than just extensions")]
+        public static GetDatabaseTypeDelegate GetDatabaseType

```

Comment:
2 items: Obsolete should wait for a future release (not suitable for a 1.43), and the getter was removed - breaking the interface here. A getter needs to be added back, as  that is used in some downstream code doing swaps.